build/makelib/helm.mk

```diff
@@ -53,6 +55,38 @@ $(HELM_INDEX): $(HELM) $(HELM_OUTPUT_DIR)
 
 helm.build: $(HELM_INDEX)
 
+# Update helm chart dependencies
+define helm.dependency.update
```

Comment:
Do we need the helm.dependency.update? I don't see it used anywhere. If it's the same as a clean, then build, can we remove this rule to keep the makefile simpler?